### PR TITLE
20760419: Candidate Profile Card Component

### PR DESCRIPTION
**Pull request by tinkabel85**

### CODE REVIEW

File: src/components/components/CandidateProfileCard/CandidateProfileCard.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import React from "react";
```
Comment (jasper-prgh) [inline src/components/components/CandidateProfileCard/CandidateProfileCard.js:1]: This is some really impressive work!

**Review by jasper-prgh — APPROVED** [review]

File: src/components/components/CandidateProfileCard/CandidateProfileCard.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import "./CandidateProfileCard.scss";
+import CandidateImage from "./assets/candidateImage.svg";
+
+function CandidateProfileCard(props) {
+  const { candidate } = props;
```
Comment (amindu00) [inline src/components/components/CandidateProfileCard/CandidateProfileCard.js]: we can get this directly by putting
function CandidateProfileCard( { candidate } ) {
in here

File: src/components/components/CandidateProfileCard/CandidateProfileCard.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import "./CandidateProfileCard.scss";
+import CandidateImage from "./assets/candidateImage.svg";
```
Comment (amindu00) [inline src/components/components/CandidateProfileCard/CandidateProfileCard.js]: i think we need to send candidate Image through the props too

**Review by amindu00 — APPROVED** [review]

File: src/components/components/CandidateProfileCard/CandidateProfileCard.js (hunk 3 above)
Comment (tinkabel85) [reply src/components/components/CandidateProfileCard/CandidateProfileCard.js]: makes sense, thank you. updated the code

File: src/components/components/CandidateProfileCard/CandidateProfileCard.js (hunk 2 above)
Comment (tinkabel85) [reply src/components/components/CandidateProfileCard/CandidateProfileCard.js]: done!

**Review by jasper-prgh — APPROVED** [review]

**Review by amindu00 — APPROVED** [review]

File: src/components/state/models/staticCandidateCards.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+export const staticCandidateCards = [
```
Comment (amindu00) [inline src/components/state/models/staticCandidateCards.js:3]: we can import candidate image to here and pass through here right?

Comment (tinkabel85) [reply src/components/state/models/staticCandidateCards.js:3]: you right) I forgot that I have created this file already

**Review by SKamesiepo — APPROVED** [review]

**Pull request merged**